vagrant/README.md

```diff
@@ -0,0 +1,12 @@
+This directory contains a [Vagrantfile](Vagrantfile) for building docker-solr.
+
+Pre-requisites: [VirtualBox](https://www.virtualbox.org/wiki/Downloads) and [Vagrant](https://www.vagrantup.com/downloads.html).
```

Comment:
Note: Latest version of vagrant does not support latest version of virtualbox, i.e. vagrant 2.2.6 needs virtualbox 6.0

Reply:
Workarounds are described on [Oracle's blog](https://blogs.oracle.com/scoter/getting-vagrant-226-working-with-virtualbox-61-ga). Others are running into this issue too, see eg [this discussion](https://github.com/oracle/vagrant-boxes/issues/178). It has been filed upstream on https://github.com/hashicorp/vagrant/issues/11249, where it will be fixed in the next release. The maintainer [commented here](https://github.com/hashicorp/vagrant/issues/11249#issuecomment-565214999) which I interpret to be soon'ish. So I don't think we need to specifically add this to the README.